### PR TITLE
Feature/data storage

### LINKED ISSUE #42: Data Management
## Fixes
- [x] Ignore 'inventaris deel' if it is a number, only keep letters
- [x] TBD: mark `OUT` pages as document, except for empty pages
- [x] Introduce `END_BEGIN` label

## Necessary improvements
- [X] Implement script for downloading inventories (`download_inventories.py`)
- [x] Implement script for annotating inventories ('segment_inventories.py`)
- [x] Implement script for storing annotated data (locally); currently all inventories are stored with label `UNK`, and annotated on the in the training script.
- [x] add thumbnail to output (see #47 )

## Perhaps; needs investigation
- [ ] Mark training and validation sets (currently dataset is shuffled upon training time)
- [ ] ~~Store data in Json format on HUC server~~
- [ ] Store annotated Jsons in separate directory

### PR DESCRIPTION
**Pull request by carschno**
Related to #42

### CODE REVIEW

**Review by carschno — COMMENTED** [review]
Double-checked.
